demo/api/src/news/generated/dto/news.sort.ts

```diff
@@ -5,7 +5,7 @@ import { Field, InputType, registerEnumType } from "@nestjs/graphql";
 import { IsEnum } from "class-validator";
 
 /* eslint-disable @typescript-eslint/naming-convention */
-// TODO: Replace with PascalCase
+// TODO: Replace with camelCase
```

Comment:
Isn't this already camel case?

Reply:
Yes, in this case. But that's generated code and for entities containing embeddables, it doesn't work because those are snake_case: 

https://github.com/vivid-planet/comet/blob/ba1b737cd844697704b6bb4c640eafcc21a8ea11/demo/api/src/products/generated/dto/manufacturer.sort.ts#L9-L20